src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int START_VALUE = 0;
+    private static final int STEP = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyValue = calculateSupplyValue(readFromFile(fromFileName));
+        int buyValue = calculateBuyValue(readFromFile(fromFileName));
+        writeInfoToFile(supplyValue, buyValue, toFileName);
```

Comment:
The method 'getStatistic' seems to be doing more than one operation. It reads from a file, calculates values, and writes to another file. Consider refactoring this method to adhere to the single responsibility principle by breaking it down into smaller methods, each with its own specific task.